src/ngSanitize/sanitize.js

```diff
@@ -262,11 +236,34 @@ function makeMap(str, lowercaseKeys) {
   return obj;
 }
 
+var baseNode;
+(function(window) {
+  var doc;
+  if (window.DOMDocument) {
+    doc = new window.DOMDocument();
```

Comment:
I do not have access to any IE, but if you already tested this with IE, then it works for me